extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -925,14 +933,14 @@ export class ProjectsController {
 		}
 	}
 
-	private getConnectionProfileFromContext(context: azdataType.IConnectionProfile | any): azdataType.IConnectionProfile | undefined {
+	private getConnectionProfileFromContext(context: azdataType.IConnectionProfile | mssqlVscode.ITreeNodeInfo | undefined): azdataType.IConnectionProfile | mssqlVscode.IConnectionInfo | undefined {
```

Comment:
The comments here aren't very clear what this "object" is that has the connectionProfile within it, but generally we should avoid using any when we can. Since I don't know the type in this case though I at least removed it from the param and then just casted it below since I assume that logic is necessary. So if anyone has context on this let me know since I would love to fix this up more properly...